src/adapter/etl-adapter-google-sheet/src/Flow/ETL/DSL/GoogleSheet.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\DSL;
+
+use Flow\ETL\Adapter\GoogleSheet\GoogleSheetExtractor;
+use Flow\ETL\Adapter\GoogleSheet\GoogleSheetRange;
+use Flow\ETL\Extractor;
+use Google\Service\Sheets;
+
+class GoogleSheet
+{
+    /**
+     * @param string $spreadsheetIds
+     * @param int $rows_in_batch
+     * @param bool $with_header
+     *
+     * @return Extractor
+     */
+    final public static function from(
+        string $spreadsheetIds,
```

Comment:
Ids? Like you can pass multiple IDS? If yes, whats the format?

Reply:
its a typo,my bad